docs/docs-content/release-notes.md

```diff
@@ -11,6 +11,18 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## November 7, 2023 - Release 4.1.9
+
+This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.
+
```

Comment:
```suggestion
## November 7, 2023 - Release 4.1.9

This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.

### Bug Fixes
```

We set the precedent to always include the header for the category. I checked because I was also unsure about this 😬  but found https://docs.spectrocloud.com/4.0.x/release-notes#august-31-2023---release-407

Reply:
@karl-cardenas-coding Oops, I forgot the heading.